tests/unit/test_decorators.py

```diff
@@ -1,58 +0,0 @@
-import numpy as np
```

Comment:
These should not be removed but changed to follow the new implementation (eg. testing that the tools.vectorize works with all relevant node types).